client/src/components/ListGroup/ListGroup.scss

```diff
@@ -55,11 +56,16 @@
 
 .list-group-item__thumbnail {
   width: 64px;
-  height: 64px;
+  height: auto;
   display: block;
   background: $gray-lighter;
   float: left;
-  margin: -12px 12px 0 #{0 - $grid-gutter-width / 2};
+  margin: -12px 12px -12px #{0 - $grid-gutter-width / 2};
+
+  > img {
+    max-height: 65px;
+    height: 100%;
```

Comment:
hmm, doesn't look stretched to me, it's possible that the thumbnail is 2x size because of retina support and could be a bug with that instead?
The second file you have is what I'm describing :)